node-mongo/infra/app/api.bicep

```diff
@@ -0,0 +1,121 @@
+param name string
```

Comment:
I wonder if putting a `service-` prefix on these service files would be worthwhile?  They would all sort together in a treeview and it sort of makes the other file name (`db-cosmos-mongo` "stick out" a little less?)